scripts/core/exp-command-buffer.yml

```diff
@@ -0,0 +1,300 @@
+--- #--------------------------------------------------------------------------
```

Comment:
> [NIT] more of a stylish thing so feel free to ignore, but I think I prefer the file name without the exp part, and we could add a comment to the file to explain that this is an exp feature.

@omarahmed1111 that's as defined in the [Contribution Guide](https://oneapi-src.github.io/unified-runtime/core/CONTRIB.html#experimental-features) and wihtout the `exp-` prefix it would be very hard to distinguish between the core and experimental features in the generator scripts.

Reply:
Ah okay, that make sense. ignore that then

Reply:
I've added the license to this file now.